Cognite.Simulator.Utils/ModelLibraryBase.cs

```diff
@@ -347,6 +347,7 @@ private async Task ExtractModelInformationAndPersist(T modelState, CancellationT
                     finally
                     {
                         await PersistModelStatus(modelState, token).ConfigureAwait(false);
+                        await PersistModelInformation(modelState, token).ConfigureAwait(false);
```

Comment:
this should go into line 345 (after `await ExtractModelInformation`) not here

Reply:
it's just weird to have model info available if `ExtractModelInformation` failed